### DISCUSSION TITLE
How to uninstall StreamFX on Windows?

### DISCUSSION THREAD

**Discussion by valandis** [Xaymar/obs-stream-effects#863] [Q&A] [answered]
Hello there,

Is there a way to completely uninstall StreamFX from OBS on Windows without damaging the OBS main installation? If yes how?

Thanks in advance!

**Comment by Xaymar** [comment, marked as answer]
The StreamFX installer will add a StreamFX uninstaller to Programs & Features. It will remove all files installed by StreamFX and leaves everything else as it is.

**Update:** The wiki now has a guide for removing StreamFX cleanly, which should work in all cases. Can't cover PEBKAC issues though.

**Reply by valandis** [reply]
Thank you for the fast response Xaymar! I had no idea I could uninstall it through "Programs and Features". Much appreciated!
